<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RibbonAnalysis</class>
 <widget class="QWidget" name="RibbonAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="Line" name="line">
        <property name="styleSheet">
         <string notr="true">QWidget {	color : #929292 ; }</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_9" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="RibbonButton" name="Descriptives">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Descriptives</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-descriptives.svg</normaloff>:/icons/analysis-descriptives.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Classical</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_10" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="RibbonButton" name="classicalTTestButton">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>T-Tests</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-classical-ttest.svg</normaloff>:/icons/analysis-classical-ttest.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="RibbonButton" name="classicalAnovaButton">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>ANOVA</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-classical-anova.svg</normaloff>:/icons/analysis-classical-anova.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="RibbonButton" name="classicalRegressionButton">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Regression</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-classical-regression.svg</normaloff>:/icons/analysis-classical-regression.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="RibbonButton" name="Crosstabs">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Crosstabs</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-classical-crosstabs.svg</normaloff>:/icons/analysis-classical-crosstabs.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_11" native="true">
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="RibbonButton" name="bayesianTTestButton">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>T-Tests</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-bayesian-ttest.svg</normaloff>:/icons/analysis-bayesian-ttest.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="RibbonButton" name="bayesianAnovaButton">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>ANOVA</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-bayesian-anova.svg</normaloff>:/icons/analysis-bayesian-anova.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="RibbonButton" name="CrosstabsBayesian">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Crosstabs</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-bayesian-crosstabs.svg</normaloff>:/icons/analysis-bayesian-crosstabs.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="RibbonButton" name="bayesianRegressionButton">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Regression</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-bayesian-regression.svg</normaloff>:/icons/analysis-bayesian-regression.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Bayesian</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QWidget" name="widget_5" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {	color : #929292 ; }</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="Line" name="line_2">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget_3</zorder>
  <zorder>widget_4</zorder>
  <zorder>widget_5</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RibbonButton</class>
   <extends>QToolButton</extends>
   <header>widgets/ribbonbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>itemSelected()</slot>
  <slot>menuItemSelected()</slot>
 </slots>
</ui>
